compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -544,40 +538,45 @@ private class SliderAdapter(
             return if (extraContentSpace == 0f) 1f else extraScrollbarSpace / extraContentSpace
         }
 
-    /**
-     * A position with cumulative offset, may be out of the container when dragging
-     */
-    var rawPosition: Float = position
-        set(value) {
-            field = value
-            position = value
-        }
-
-    /**
-     * Actual scroll of content regarding slider layout
-     */
-    private var scrollPosition: Float
+    private var rawPosition: Float
         get() = scrollScale * adapter.scrollOffset
         set(value) {
             runBlocking {
                 adapter.scrollTo(containerSize, value / scrollScale)
             }
         }
 
-    /**
-     * Actual position of a thumb within slider container
-     */
     var position: Float
-        get() = if (reverseLayout) containerSize - size - scrollPosition else scrollPosition
+        get() = if (reverseLayout) containerSize - size - rawPosition else rawPosition
         set(value) {
-            scrollPosition = if (reverseLayout) {
+            rawPosition = if (reverseLayout) {
                 containerSize - size - value
             } else {
                 value
             }
         }
 
     val bounds get() = position..position + size
+
+    // Stores the unrestricted position during a dragging gesture
+    private var positionDuringDrag = 0f
+
+    // Called when the thumb dragging starts
+    fun onDragStarted() {
+        positionDuringDrag = position
+    }
+
+    // Called on every movement while dragging the thumb
```

Comment:
```suggestion
    /**
     * Called on every movement while dragging the thumb
     */
```

It helps to use "Quick Documentation" action to understand function behaviour.

<img width="550" alt="image" src="https://user-images.githubusercontent.com/99798741/210519918-20170d12-805d-4aa8-b725-4d0c81997fb9.png">

Reply:
Changed.